jubilant/_juju.py

```diff
@@ -237,6 +238,21 @@ def add_secret(
 
         return SecretURI(output.strip())
 
+    def add_ssh_key(self, *ssh_keys: str) -> None:
```

Comment:
Elsewhere in Jubilant where there are 0-n or 1-n arguments, we use `str | Iterable[str]` or `str | Iterable[str] | None`. I think we should follow that pattern here, even though in many of the other cases it's something joined with a comma.

Let's make the signature `def add_ssh_key(self, ssh_key: str | Iterable[str]) -> None:` and adjust the code to match.

We should do this same change for `remove_ssh_key` as well.